packages/liquidsoap/liquidsoap.2.1.0/opam

```diff
@@ -0,0 +1,186 @@
+opam-version: "2.0"
+synopsis: "Swiss-army knife for multimedia streaming"
+description: """\
+Liquidsoap is a powerful and flexible language for describing your
+streams. It offers a rich collection of operators that you can combine
+at will, giving you more power than you need for creating or
+transforming streams. But liquidsoap is still very light and easy to
+use, in the Unix tradition of simple strong components working
+together."""
+maintainer: "romain.beauxis@gmail.com"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+license: "GPL-2.0-or-later"
+homepage: "https://github.com/savonet/liquidsoap"
+bug-reports: "https://github.com/savonet/liquidsoap/issues"
+depends: [
+  "conf-autoconf" {dev}
+  "conf-automake" {dev}
+  "ocaml" {>= "4.12.0" & <= "5.0.0"}
```

Comment:
Did you mean including 5.0 or excluding 5.0? If it’s excluded, `<=` should be avoided as it will not include minor releases
```suggestion
  "ocaml" {>= "4.12.0" & < "5.0.0"}
```